sdk/nodejs/runtime/rpc.ts

```diff
@@ -394,7 +394,7 @@ export function deserializeProperty(prop: any): any {
         throw new Error("unexpected undefined property value during deserialization");
     }
     else if (prop === unknownValue) {
-        return undefined;
+        return isDryRun() ? unknown : undefined;
```

Comment:
ok, you needed this value for testing.  in that case, export it, but mark it `@internal` so it won't be listed in our d.ts

Reply:
Done.